### PR TITLE
[4.0] Language switcher: improving display of dropdown with flags

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Adding shadow to dropdown. Normalizing RTL and LTR. Using em instead of px.
Taking off useless `Reduce Line Height` parameter (as we now use bootstrap).
Correcting arrow distance from dropdown.

### Testing Instructions
Install multilingual site with sample data, including Persian language.
I used `فارس  (Iran)` for Persian native content language)
Edit mod_languages settings to:

Screen Shot 2021-04-09 at 18 37 11

Display the switcher in frontend
Patch and test.
(`Reduce Line Height` will not display any more in module params)

### Actual result BEFORE applying this Pull Request
LTR

before_ltr

RTL

before_rtl

### Expected result AFTER applying this Pull Request
LTR

after_ltr

RTL

after_rtl

### CODE REVIEW

File: build/media_source/mod_languages/css/template.css [hunk 1]
```diff
@@ -3,24 +3,54 @@ div.mod-languages ul {
...
 div.mod-languages a {
 	text-decoration: none;
 }
+
+div.mod-languages .btn-group {
+	display: flex !important;
+}
+
+div.mod-languages .btn-group .btn {
+	flex: none;;
```
Comment (brianteeman) [inline build/media_source/mod_languages/css/template.css]:
```suggestion
	flex: none;
```

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 667e6959b0f4f44351996277fb8706e0fb538bce<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33082">issues.joomla.org/tracker/joomla-cms/33082</a>.</sub>

**Comment by pabloarias** [comment]
I have tested this item :red_circle: unsuccessfully on 667e6959b0f4f44351996277fb8706e0fb538bce<br /><br />All is working fine but the alignment of languages in the dropdown in RTL language. They are aligned to left.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33082">issues.joomla.org/tracker/joomla-cms/33082</a>.</sub>

**Comment by infograf768** [comment]
@pabloarias 
Maybe you did not use the correct clean install with patch as I get here OK
Screen Shot 2021-04-28 at 10 17 04

I suggest you make a clean install with te pre-built packages:
https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/33082/downloads/41791/

**Comment by pabloarias** [comment]
I have tested this item :white_check_mark: successfully on 667e6959b0f4f44351996277fb8706e0fb538bce<br /><br />You are right, after installing the package with Joomla and the PR it works. Thank you!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33082">issues.joomla.org/tracker/joomla-cms/33082</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33082">issues.joomla.org/tracker/joomla-cms/33082</a>.</sub>

**Pull request merged**